GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1164,6 +1164,8 @@ string BuildPathFilter(string? path)
                     $"--format=\"{startOfObjectId}%H\"",
                     "--name-only",
                     "--follow",
+                    "--all",
+                    "-m",
```

Comment:
Probably need `--diff-merges=separate` too, if the user do not use default config
https://git-scm.com/docs/git-log#Documentation/git-log.txt--m